dev/bots/test.dart

```diff
@@ -417,7 +417,7 @@ Future<void> _pubRunTest(
   bool enableFlutterToolAsserts = false,
   bq.TabledataResourceApi tableData,
 }) async {
-  final List<String> args = <String>['run', 'test', useFlutterTestFormatter ? '-rjson' : '-rcompact', '-j1'];
+  final List<String> args = <String>['run', 'test', if (useFlutterTestFormatter) '-rjson' else '-rcompact', '-j1'];
```

Comment:
I'm not a big fan of if-else instead of conditional expression. However I think the lines below should be include in the list creation.

```dart
final List<String> args = <String>[
  'run', 
  'test', 
  if (useFlutterTestFormatter) '-rjson' else '-rcompact',
  '-j1',
  if (!hasColor) '--no-color',
  if (testPath != null) testPath,
];

Reply:
yeah i don't think the inline `if/else` is better than `?:`. It's fine when there's no `else`, as below (the ones @a14n lists).